README.md

```diff
@@ -14,6 +14,7 @@ ___
 * RefApp 2.x Workflow Tests (Chrome) [BDD Chrome](https://ci.openmrs.org/browse/CONTRIB-QA) ![Build Status](https://ci.openmrs.org/plugins/servlet/wittified/build-status/CONTRIB-QA)
   * [![RefApp 2.x Clinical Visit Test](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-clinical-visit.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-clinical-visit.yml)
   * [![RefApp 2.x Vitals And Triaging Test](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-vitals-and-triaging.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-vitals-and-triaging.yml) 
+  * [![RefApp 2.x Manage User Account Test](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-manage-user-account.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-manage-user-account.yml)
   * Tests in development:
     * Managing User Accounts
```

Comment:
Remove this line 19

Reply:
Manage user account workflow is under development the `readme` file stipulates. We will  always update the readme file based on which workflow is done.

Reply:
Why would you still remain it in readme when its the one you are working on then, that means if your workflow gets merged, you will then create another pull request for removing the tests in development!, i see this as wasting resources, Lets try to always include them in the ticket that is ready

Reply:
Am concurrently working on [RATEST-157](https://issues.openmrs.org/browse/RATEST-180), [RATEST-180](https://issues.openmrs.org/browse/RATEST-180)  and I updated the readme file in the commit for RATEST-180

Reply:
Collect the commits , i think you were to remove the manage accounts in readme with in this file, it wont track the workflow and commit history very clearly.

Reply:
As mentioned [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/78#discussion_r692186811) am working on two workflows and opted to update the readme file at once. BTW its a week ago when we decided to temporarily include a few of the E2E workflow tests that are under development but we will in the future not include them in the readme file since they are being tracked under our Jira Dashboard.

Reply:
i think you need to only track changes where application and meaningfull